apps/bl-ap-imgproc.bash

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+export PYTHONUNBUFFERED=yes
+
+# CPU usage estimate (ps aux):
+
+/usr/local/bin/sirius-ioc-bl-ap-imgproc.py --devname CAX:B:BASLER01 \
```

Comment:
Maybe use one more group here to indicate these PVs are from this IOC, example: CAX:B:BASLER01:HLImgProc